service/ztests/csv-error.yaml

```diff
@@ -1,20 +0,0 @@
-script: |
```

Comment:
How about skipping all these tests instead of deleting them?

Reply:
Sure